benchopt/benchmark.py

```diff
@@ -5,6 +5,7 @@
 import importlib
 import itertools
 from pathlib import Path
+import matplotlib.pyplot as plt
```

Comment:
If possible, let's avoid this import.
Importing matplotlib is slow (almost 200ms), and this slows down completion. (I know it is a bit extreme but for completion, it is always chasing after any unecessary imports).

Better to import it only in the function when it is needed.